theseus/src/api/profile.rs

```diff
@@ -71,6 +68,28 @@ pub async fn get(
     Ok(profile)
 }
 
+/// Gets all projects version metadata for a profile
+/// Allows loading metadata without loading all project data
+#[tracing::instrument]
+pub async fn get_projects_metadata(
+    path: &Path,
+) -> crate::Result<Option<HashMap<PathBuf, ProjectMetadata>>> {
```

Comment:
what is this used for?

Reply:
See below comment for "the instance projects are already available"